materials/paper_list/FL-Tree/README.md

```diff
@@ -1,27 +1,29 @@
-# Federated Learning for Tree
+## Federated Learning for Tree
```

Comment:
Maybe we should change it to "Federated Learning for Tree-based Models"

Reply:
fine